authority/admin/api/acme.go

```diff
@@ -0,0 +1,113 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/smallstep/certificates/api"
+	"github.com/smallstep/certificates/authority/admin"
+	"go.step.sm/linkedca"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// CreateExternalAccountKeyRequest is the type for POST /admin/acme/eab requests
+type CreateExternalAccountKeyRequest struct {
+	ProvisionerName string `json:"provisioner"`
+	Name            string `json:"name"`
+}
+
+// Validate validates a new-admin request body.
+func (r *CreateExternalAccountKeyRequest) Validate() error {
+	if r.ProvisionerName == "" {
+		return admin.NewError(admin.ErrorBadRequestType, "provisioner name cannot be empty")
+	}
+	if r.Name == "" {
+		return admin.NewError(admin.ErrorBadRequestType, "name / reference cannot be empty")
+	}
+	return nil
+}
+
+// GetExternalAccountKeysResponse is the type for GET /admin/acme/eab responses
+type GetExternalAccountKeysResponse struct {
+	EAKs       []*linkedca.EABKey `json:"eaks"`
+	NextCursor string             `json:"nextCursor"`
+}
+
+// CreateExternalAccountKey creates a new External Account Binding key
+func (h *Handler) CreateExternalAccountKey(w http.ResponseWriter, r *http.Request) {
+	var body CreateExternalAccountKeyRequest
+	if err := api.ReadJSON(r.Body, &body); err != nil {
+		api.WriteError(w, admin.WrapError(admin.ErrorBadRequestType, err, "error reading request body"))
+		return
+	}
+
+	if err := body.Validate(); err != nil {
+		api.WriteError(w, err)
+		return
+	}
+
+	eak, err := h.acmeDB.CreateExternalAccountKey(r.Context(), body.ProvisionerName, body.Name)
```

Comment:
Why are EAKs associated with a provisioner? I'm not asking because I think they shouldn't be, just want to make sure I understand.